CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2021-02-25
+### Breaking Changes
+- Updated tokio to v1, this is a breaking change when using coldsnap as a library. [#39]
```

Comment:
Did the tokio bump not require any coldsnap code changes?

Reply:
It did not. But it should be called out with a 'major' version bump and as a breaking change since a lib user needs to coordinate all their deps to tokio v1 at the same time.

Reply:
Oh wait, actually it did have some tiny tweaks. #39